show/main.py

```diff
@@ -668,6 +668,48 @@ def alias(interfacename):
                 body.append([port_name, port_name])
 
     click.echo(tabulate(body, header))
+    
```

Comment:
Same comment here as in 'config'
A correct implementation can be:
* show txmon status
To show the current interval and then a table of all configured interfaces.

* show txmon interval
* show txmon interfaces